code/game/machinery/stasis.dm

```diff
@@ -158,12 +158,13 @@
 	. = ..()
 	return default_deconstruction_crowbar(I) || .
 
-/obj/machinery/stasis/nap_violation(mob/violator)
-	unbuckle_mob(violator, TRUE)
-
+//for some reason, in testing, cyborgs were unable to unbuckle people from stasis beds unless this extension of attack_robot() here was present, so I guess it stays in
 /obj/machinery/stasis/attack_robot(mob/user)
 	if(Adjacent(user) && occupant)
 		unbuckle_mob(occupant)
 	else
 		..()
+
+/obj/machinery/stasis/nap_violation(mob/violator)
```

Comment:
why did you put nap violation code there

Reply:
fix this please, it interferes with the blame for no reason

Reply:
Is the blame fixed now? I can't tell.

Reply:
it LOOKS okay to me

lemme just add another enter at the end so the end of the file looks better

Reply:
@ExcessiveUseOfCobblestone did I do this right?